### ISSUE TITLE
CSS issue when used along with bootstrap.

### ISSUE THREAD

**Issue by Vickyraz** [stephanrauh/ngx-extended-pdf-viewer#783]
Hi,
I am having an issue when using ngx-extended-pdf-viewer in a project along with bootstrap.min.js.
I have included the following cdn for bootstrap css in index.html. (With out this, there is no issue.)

`  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" rel="stylesheet">`

**Issue screenshot :**

![image](https://user-images.githubusercontent.com/17909417/120339528-d11b7f80-c312-11eb-8c7a-201f38c9bbc3.png)

**Version info**
Library : Latest version - 9.0.0-rc.3
Angular : Checked in Angular 9 and Angular 10

Browser : Chrome (Version 91.0.4472.77), Edge (Version 90.0.818.62)

**To Reproduce**
Create a sample project with ngx-extended-pdf-viewer.
Add the following code in index.html
`  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" rel="stylesheet">`

Thanks,
Raz

**Comment by stephanrauh** [comment]
What's the error precisely? I can see the sidebar looks a bit crammed, and I can guess the spacing between two consecutive pages is missing. Is this correct?

**Comment by stephanrauh** [comment]
Turns out a single CSS rule fixes the bug. I can do that, but not today. If you're in a hurry - the final release is just a few days away - add this CSS rule to your global `styles.css` or `styles.scss` file:

```CSS
* {
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
}
```

**Comment by Vickyraz** [comment]
> What's the error precisely? I can see the sidebar looks a bit crammed, and I can guess the spacing between two consecutive pages is missing. Is this correct?

Yes. Those are the issues I noted.

**Comment by Vickyraz** [comment]
> Turns out a single CSS rule fixes the bug. I can do that, but not today. If you're in a hurry - the final release is just a few days away - add this CSS rule to your global `styles.css` or `styles.scss` file:
> 
> ```css
> * {
>   -webkit-box-sizing: content-box;
>   -moz-box-sizing: content-box;
>   box-sizing: content-box;
> }
> ```

This fixes the issue. Thanks @stephanrauh

**Comment by stephanrauh** [comment]
I've just shipped version 9.0.0.
